test/end-to-end/helpers/index.js

```diff
@@ -0,0 +1,71 @@
+'use strict';
+
+const Chance = require('chance');
+const querystring = require('querystring');
+
+const chance = new Chance();
+
+const helpers = {};
+
+helpers.getAuthClientParams = function() {
+  return {
+    clientId: 'e922556a-7d4f-4168-88cd-059276044798',
```

Comment:
It was my bad for pushing these up, but you should make these environment variables. Ask @RogWilco about which client id and secret to use. I've rolled the secret for this so no one can use it.